### PR TITLE
Add support to provide date format to return UTC timestamp

### PR DESCRIPTION
**Pull request by dnwick**
## Purpose
Provide more flexibility to the function

## Goals
Increase user flexibility

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

### CODE REVIEW

File: component/src/main/java/io/siddhi/extension/execution/time/UTCTimestampFunctionExtension.java [hunk 1]
```diff
@@ -48,31 +50,65 @@
 @Extension(
         name = "utcTimestamp",
         namespace = "time",
-        description = "Function returns the system current time in UTC timezone with `yyyy-MM-dd HH:mm:ss` format.",
+        description = "Function returns the system current time in UTC timezone with given date format which " +
+                "defaults to `yyyy-MM-dd HH:mm:ss` if not provided.",
+        parameters = {
+                @Parameter(name = "date.format",
+                        description = "The format of the date value provided. " +
+                                "For example, `yyyy/MM/dd HH:mm:ss.SSS`.",
```
Comment (tishan89) [inline component/src/main/java/io/siddhi/extension/execution/time/UTCTimestampFunctionExtension.java:59]: Do we support yy/MM/dd. Else lts clearly state it here

File: component/src/test/java/io/siddhi/extension/execution/time/UTCTimestampFunctionExtensionTestCase.java [hunk 2]
```diff
@@ -83,4 +83,42 @@ public void receive(long timeStamp, Event[] inEvents, Event[] removeEvents) {
...
+        });
+
+        InputHandler inputHandler = executionPlanRuntime.getInputHandler("inputStream");
+        executionPlanRuntime.start();
+        inputHandler.send(new Object[]{"IBM", 700f, "yyyy-MM-dd HH:mm:ss.SSS"});
+        inputHandler.send(new Object[]{"WSO2", 60.5f, "yyyy-MM-dd HH:mm:ss"});
+        inputHandler.send(new Object[]{"XYZ", 60.5f, "yyyy-MM-dd HH:mm"});
+        SiddhiTestHelper.waitForEvents(waitTime, 3, eventCount, timeout);
+        AssertJUnit.assertEquals(3, eventCount.get());
+        AssertJUnit.assertTrue(eventArrived);
```
Comment (tishan89) [inline component/src/test/java/io/siddhi/extension/execution/time/UTCTimestampFunctionExtensionTestCase.java:141]: Assert that you are getting in the format you requested using a regex

**Review by tishan89 — CHANGES_REQUESTED** [review]

**Review by sajithshn — APPROVED** [review]

File: component/src/main/java/io/siddhi/extension/execution/time/UTCTimestampFunctionExtension.java (hunk 1 above)
Comment (dnwick) [reply component/src/main/java/io/siddhi/extension/execution/time/UTCTimestampFunctionExtension.java:59]: yes its any format user provides basically

File: component/src/test/java/io/siddhi/extension/execution/time/UTCTimestampFunctionExtensionTestCase.java (hunk 2 above)
Comment (dnwick) [reply component/src/test/java/io/siddhi/extension/execution/time/UTCTimestampFunctionExtensionTestCase.java:141]: fixed in https://github.com/siddhi-io/siddhi-execution-time/pull/62/commits/bbc32f142194c2241f35bd972af3ff0b75661578

**Review by tishan89 — APPROVED** [review]

**Pull request merged**